crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -37,16 +162,18 @@ pub enum AlgorithmInfo {
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct EncryptionInfo {
     /// The user ID of the event sender, note this is untrusted data unless the
-    /// `verification_state` is as well trusted.
+    /// `verification_state` is `Verified` as well.
     pub sender: OwnedUserId,
     /// The device ID of the device that sent us the event, note this is
-    /// untrusted data unless `verification_state` is as well trusted.
+    ///  untrusted data unless `verification_state` is `Verified` as well.
     pub sender_device: Option<OwnedDeviceId>,
     /// Information about the algorithm that was used to encrypt the event.
     pub algorithm_info: AlgorithmInfo,
     /// The verification state of the device that sent us the event, note this
     /// is the state of the device at the time of decryption. It may change in
     /// the future if a device gets verified or deleted.
+    /// Callers that persist this should mark the state as dirty when a device
+    /// change is received down the sync.
```

Comment:
```suggestion
    /// the future if a device gets verified or deleted.
    ///
    /// Callers that persist this should mark the state as dirty when a device
    /// change is received down the sync.
```

Reply:
Markdown paragraph because otherwise everything will be reflowed into a single paragraph, which doesn't appear to be the intention.